index.html

```diff
@@ -2,130 +2,406 @@
-    .white {}
+            .chessboard .row::after {
+                left: 100%;
+            }
+            .chessboard .rank-8::before,
+            .chessboard .rank-8::after {
+                content: '8';
+                position: absolute;
+            }
 
-    .rook {}
+            .chessboard .rank-7::before,
+            .chessboard .rank-7::after {
+                content: '7';
+            }
+            .chessboard .rank-6::before,
+            .chessboard .rank-6::after {
+                content: '6';
+            }
 
-    .piece-white-rook {
-      /* What goes here? */
-    }
-  </style>
-  <div>
-    <main class="container">
+            .chessboard .rank-5::before,
+            .chessboard .rank-5::after {
+                content: '5';
+            }
+            .chessboard .rank-4::before,
+            .chessboard .rank-4::after {
+                content: '4';
+                position: absolute;
+            }
+
+            .chessboard .rank-3::before,
+            .chessboard .rank-3::after {
+                content: '3';
+            }
+            .chessboard .rank-2::before,
+            .chessboard .rank-2::after {
+                content: '2';
+            }
+
+            .chessboard .rank-1::before,
+            .chessboard .rank-1::after {
+                content: '1';
+            }
+            /* END Row Styles */
+
+            .piece {
+              font-size: 4.5em;
+              padding-left: 15px;
+            }
 
-      <style>
-
-      </style>
-      <div class="chessboard">
-        <ul class="row rank-2">
-          <li class="square file-a">
-            <!-- single square -->
-          </li>
-          <li class="square file-b"></li>
-        </ul>
-        <!-- .row -->
-        <ul class="row rank-1">
-          <li class="square file-a">&#x2656;</li>
-          <li class="square file-b"></li>
-
-        </ul>
-        <!-- .row -->
-      </div>
-      <!-- .chessboard -->
+            .buttons_wrap {
+              margin: 0 auto;
+              width: 215px;
+              padding-top: 20px;
+            }
+
+            button {
+              display: inline-block;
+              background-color: #BEDB39;
+              border: none;
+              border-radius: 5px;
+              padding: 7px;
+              font-size: 25px;
+            }
+
+            button:hover {
+              background-color: #96B01E;
+            }
+
+            /* @media queries! */
+            @media screen and (max-width: 768px) {
+              .container {
+              max-width: 600px;
+    		      font-size: 50%;
+        	    }
+              li.letters {
+                padding: 0 4.9%;
+              }
+            }
+            @media screen and (max-width: 400px) {
+              .container {
+    		        font-size: 50%;
+        	    }
+              .piece {
+                padding-top: 10px;
+                padding-left: 20px;
+              }
+            }
+        </style>
```

Comment:
Considering how much CSS is inside this `<style>` tag, you're _way_ past the point of a necessary refactor. When you write more than 10 lines of CSS that does _something_, stop writing and refactor: start by moving styles into the CSS file, then look for duplication or repetition to remove.